website/static/js/file-browser.js

```diff
@@ -0,0 +1,284 @@
+            ]),
+            m('.fb-main', { config: ctrl.updateList, style : poStyle },
+                m('#poOrganizer', m('.spinner-loading-wrapper', m('.logo-spin.logo-md')))
+            ),
+            infoPanel
+        ]);
+    }
+};
+
+/**
+ * Collections Module.
+ * @constructor
+ */
+var Collections  = {
+    controller : function (args) {
+        var self = this;
+        self.updateCollection = function(){
+           args.updateCollection(this);
+        };
+    },
+    view : function (ctrl, args) {
+        var selectedCSS;
+        return m('.fb-collections', m('ul', [
+            args.list.map(function(item, index, array){
+                selectedCSS = item.id === args.activeCollection() ? '.active' : '';
+                return m('li' + selectedCSS,
+                    m('a', { href : '#', onclick : ctrl.updateCollection.bind(item) },  item.label)
+                );
+            })
+        ]));
+    }
+};
+
+/**
+ * Breadcrumbs Module.
+ * @constructor
+ */
+var Breadcrumbs = {
+    controller : function (args) {
+        var self = this;
+        self.updateFilesData = function() {
+            args.updateFilesData(this);
+        };
+    },
+    view : function (ctrl, args) {
+        return m('.fb-breadcrumbs', m('ul', [
+            args.data().map(function(item, index, array){
+                if(index === array.length-1){
+                    return m('li',  item.label);
+                }
+                var linkObject = {
+                    type : 'breadcrumb',
+                    data : item,
+                    label : item.label,
+                    id : null,
+                    index : index
+                };
+                return m('li',
+                    m('a', { href : '#', onclick : ctrl.updateFilesData.bind(linkObject)},  item.label),
+                    m('i.fa.fa-chevron-right')
+                );
+            })
+
+        ]));
+    }
+};
+
+
+/**
+ * Filters Module.
+ * @constructor
+ */
+var Filters = {
+    controller : function (args) {
+        var self = this;
+        self.updateUser = function(){
+            args.updateUser(this);
+        };
+    },
+    view : function (ctrl, args) {
+        var selectedCSS;
+        return m('.fb-filters.m-t-lg',
+            [
+                m('h4', 'Filters'),
+                m('', 'Contributors'),
+                m('ul', [
+                    args.nameFilters.map(function(item){
+                        selectedCSS = item.id === args.activeUser() ? '.active' : '';
+                        return m('li' + selectedCSS,
+                            m('a', { href : '#', onclick : ctrl.updateUser.bind(item)}, item.label)
+                        );
+                    })
+                ])
+
+            ]
+        );
+    }
+};
+
+/**
+ * Information Module.
+ * @constructor
+ */
+var Information = {
+    view : function (ctrl, args) {
+        var template = '';
```

Comment:
Again, it looks like `args` is expected to be an instance of some class and could probably be named better.